evaluate.js

```diff
@@ -0,0 +1,85 @@
+Array.prototype.clean = function() {
```

Comment:
in my opinion you do not need this one
use build-in array filter method

Reply:
changed as recommended